recipes/hole-filling-liepa/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "hole-filling-liepa" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: {{ version }}
+
+source:
+  url: https://github.com/russelmann/hole-filling-liepa/archive/v{{ version }}.tar.gz
+  sha256: c73a65bd39b30801f5c85f7245de0f1ae8458d6eb78d657f63d8976d63cf4a47
+
+build:
+  number: 0
+  skip: true  # [py<38]
```

```suggestion
  skip: True  # [py<38]
```
I think this capitalization matters??? Even though it's moot because we don't build 37 on conda-forge anymore.